index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
```

Comment:
Can this file go in a more appropriate place than the project root?

Maybe under `reader-gtfs` somewhere since the readme describing it is there?